test/integration/trezor/node/votingRegistration.js

```diff
@@ -45,7 +45,9 @@ async function testVotingRegistrationMetaDataSigning(cryptoProvider, votingRegis
 
 describe('Trezor sign voting registration metadata', () => {
   let cryptoProvider
-  before(async () => {
+  // eslint-disable-next-line func-names
+  before(async function () {
+    this.timeout(10000)
     cryptoProvider = await TrezorCryptoProvider()
   })
   after(async () => {
```

Comment:
The after function shouldn't actually be needed anymore if you add `--exit` to the `test-integration-trezor` package.json script. Then both `tx.js` and `votingRegistration.js` tests can run sequentially.

Reply:
Works :+1: